UnityProject/Assets/Scripts/HealthV2/Living/PolymorphicSystems/Bodypart/ReagentCirculatedComponent.cs

```diff
@@ -0,0 +1,21 @@
+using System.Linq;
+using UnityEngine;
+
+namespace HealthV2.Living.PolymorphicSystems.Bodypart
+{
+	public class ReagentCirculatedComponent : BodyPartComponentBase
+	{
+		[Tooltip(" doesn't contribute to the volume of the blood pool, Just used for How much bleeding happens That type of thing")]
+		public float Throughput;
+
+		public override HealthSystemBase GenSystem(LivingHealthMasterBase livingHealth)
+		{
+			return new ReagentPoolSystem();
+		}
+
+		public override bool HasSystem(LivingHealthMasterBase livingHealth)
+		{
+			return livingHealth.ActiveSystems.OfType<ReagentPoolSystem>().Any();
```

Comment:
Wouldn't this always return true? Also, using LINQ and ofType.. how often is this function called? Because it would be bad for GC and performance if its frequently called.

Reply:
when body parts are added and removed, that's when it's checked